{
	"id": "SUSE-SU-2020:2109-1",
	"modified": "2020-08-03T14:47:14Z",
	"published": "2020-08-03T14:47:14Z",
	"related": [
		"CVE-2020-14019"
	],
	"summary": "Security update for python-rtslib-fb",
	"details": "This update for python-rtslib-fb fixes the following issues:\n\n- Update to version v2.1.73 (bsc#1173257 CVE-2020-14019):\n  * version 2.1.73\n  * save_to_file: fix fd open mode\n  * saveconfig: copy temp configfile with permissions\n  * saveconfig: open the temp configfile with modes set\n  * Fix 'is not' with a literal SyntaxWarning\n  * Fix an incorrect config path in two comments\n  * version 2.1.72\n  * Do not change dbroot after drivers have been registered\n  * Remove '_if_needed' from RTSRoot._set_dbroot()'s name\n  Replacing old tarball with python-rtslib-fb-v2.1.73.tar.xz\n",
	"affected": [
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Module for Basesystem 15 SP2",
				"name": "python-rtslib-fb",
				"purl": "pkg:rpm/suse/python-rtslib-fb&distro=SUSE%20Linux%20Enterprise%20Module%20for%20Basesystem%2015%20SP2"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.1.73-3.3.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-rtslib-fb": "2.1.73-3.3.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Module for Python 2 15 SP2",
				"name": "python-rtslib-fb",
				"purl": "pkg:rpm/suse/python-rtslib-fb&distro=SUSE%20Linux%20Enterprise%20Module%20for%20Python%202%2015%20SP2"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.1.73-3.3.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python2-rtslib-fb": "2.1.73-3.3.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2020/suse-su-20202109-1/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1173257"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2020-14019"
		}
	]
}
